app/controllers/public/api/v1/inboxes/conversations_controller.rb

```diff
@@ -30,7 +46,7 @@ def update_last_seen
   private
 
   def set_conversation
-    @conversation = @contact_inbox.contact.conversations.find_by!(display_id: params[:id])
+    @conversation = @contact_inbox.conversations.find_by!(display_id: params[:id])
```

Comment:
not to break existing workflows and also to stay consistent with `index` method

```suggestion
    if @contact_inbox.hmac_verified? 
       @contact_inbox.contact.conversations.find_by!(display_id: params[:id])
    else
       @contact_inbox.conversations.find_by!(display_id: params[:id])
     end
```